Jenkinsfile

```diff
@@ -81,6 +81,8 @@ node {
     def clustername = "jenkins-" + sh([returnStdout: true, script: '''openssl rand \
         -base64 100 | tr -dc a-z0-9 | cut -c -25''']).trim()
 
+    def version = sh([returnStdout: true, script: 'git describe --always --abbrev=7 --dirty']).trim()
```

Comment:
I think the `--dirty` poses a problem. I see [here](https://service-catalog-jenkins.appspot.com/job/service-catalog-PR-testing2/295/console) that the image tag includes `-dirty` in it, and that image of course doesn't exist.